src/transaction/components/TransactionProvider.test.tsx

```diff
@@ -309,7 +315,12 @@ describe('TransactionProvider', () => {
     );
     fireEvent.click(button);
     await waitFor(() => {
-      expect(onErrorMock).toHaveBeenCalled();
+      expect(sendWalletTransactionsMock).toHaveBeenCalled();
+      expect(onErrorMock).toHaveBeenCalledWith({
+        code: 'TmTPc01',
+        error: '{}',
+        message: 'Something went wrong. Please try again.',
+      });
```

Comment:
nit: could use expect.objectContaining here to only test pertinent parts of error